src/main/java/frc/robot/controllers/Climber.java

```diff
@@ -92,9 +95,16 @@ public void loop() {
     }
 
     public boolean isClimbDone() {
-        double encoderValue = telescopeEncoderMotor.getSelectedSensorPosition();
-        if (encoderValue < 20) {
-            return true;
+        if (desiredPosition == topEncoderHeight) {
+            if ((currentPosition >= topEncoderHeight - 5) && (currentPosition <= topEncoderHeight + 5)) {
```

Comment:
Magic numbers are no fun. I don't know what 5 means. Please change to a variable with a self explanatory name.